Makefile

```diff
@@ -79,6 +79,9 @@ GREEN  := $(shell tput -Txterm setaf 2)
 YELLOW := $(shell tput -Txterm setaf 3)
 RESET  := $(shell tput -Txterm sgr0)
 
+OUTPUT=$(shell grep -e 'webhookdefinitions' /home/ves/Desktop/demo/tmp/addon-operator/bundle/manifests/addon-operator.clusterserviceversion.yaml)
```

Comment:
This points to your local fork (`/home/ves`) :smile: 
Better to use `abspath` instead